pkg/targets/adapter/splunktarget/adapter.go

```diff
@@ -82,13 +90,21 @@ func NewTarget(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClien
 		logger.Panicw("Invalid HEC endpoint URL "+env.HECEndpoint, zap.Error(err))
 	}
 
+	replier, err := targetce.New(env.Component, logger.Named("replier"),
+		targetce.ReplierWithStatefulHeaders(env.BridgeIdentifier),
+		targetce.ReplierWithStaticResponseType(v1alpha1.EventTypeSplunkResponse),
+		targetce.ReplierWithPayloadPolicy(targetce.PayloadPolicy(env.CloudEventPayloadPolicy)))
+	if err != nil {
+		logger.Panicf("Error creating CloudEvents replier: %v", err)
+	}
```

Comment:
Will this panic if `EVENTS_PAYLOAD_POLICY` isn't set?
I see the controller only sets it conditionally.

Reply:
No. It does not. same for `env.CloudEventPayloadPolicy`